### PR TITLE
add back-to-top component on pages: home, product and search

### PR DESCRIPTION
**Pull request by alexandreTrino**
#### What problem is this solving?

Added the button to back-to-top on Homepage, Search and Product

Improve UI Theme

#### How should this be manually tested?

Scroll page down Homepage, Search and Product

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [ ] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [x] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Screenshots or example usage

https://trino--vtexmx.myvtex.com/

#### Type of changes

✔️ | Type of Change
---|---
_ | New feature

### CODE REVIEW

**Comment by alexandreTrino** [comment]
I forgot to change patch version on `Manifest.json` XD

**Comment by victorhmp** [comment]
Hi @alexandreTrino ! Thanks for the contribution! We’ll review this on Monday :)

Also, please don’t forget to add Workspace with this version of the app linked in the description of this PR so that we can see how it looks.

**Comment by victorhmp** [comment]
> I forgot to change patch version on `Manifest.json` XD

Oh! There shouldn’t be any changes in the `manifest.json` in the PR. The release process is automated by a bot :)

**Comment by alexandreTrino** [comment]
WORKSPACE
--------------

`https://trino--vtexmx.myvtex.com/`

File: store/blocks/search.jsonc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "store.search": {
-    "blocks": ["search-result-layout"],
+    "blocks": ["search-result-layout","back-to-top-button"],
```
Comment (klzns) [inline store/blocks/search.jsonc:3]:
```suggestion
    "blocks": ["search-result-layout", "back-to-top-button"],
```

File: store/blocks/search.jsonc [hunk 2]
```diff
@@ -226,5 +226,13 @@
 
   "gallery": {
     "blocks": ["product-summary.shelf"]
+  },
+
+  "back-to-top-button":{
+    "props":{
+      "displayThreshold": 800,
+      "display":"caret-icon"
+    }
```
Comment (klzns) [inline store/blocks/search.jsonc:235]: Block definitions are shared across files, so you only need to define a block once.

File: store/blocks/product.jsonc [hunk 3]
```diff
@@ -136,5 +137,12 @@
         "Pinterest": true
       }
     }
+  },
+
+  "back-to-top-button":{
+    "props":{
+      "displayThreshold": 800,
+      "display":"caret-icon"
+    }
```
Comment (klzns) [inline store/blocks/product.jsonc:146]: Remove, same as other comment

File: store/blocks/search.jsonc (hunk 2 above)
Comment (klzns) [reply store/blocks/search.jsonc:235]: We get all JSON files and compile it to one big JSON.

**Comment by iaronaraujo** [comment]
Closing due to inactivity! Feel free to open this up again!

**Pull request closed without merging**